### DISCUSSION TITLE
Wrong type of activity on Strava

### DISCUSSION THREAD

**Discussion by QQ7ita** [mytourbook/mytourbook#1134] [General]
HI, I have the last release of mtb 23.5.0, as usual after having assigned all tags and typo of activity in mtb I upload the activity to Strava (free). The last 2-3 activities I have uploaded are signed on Strava as General Training (I have the italian language and I don't know how exactly it is shown in english) instead of Cycling as he always did until a few days ago.
Can it depend on mtb or is it a Strava problem?
Thanks
Flavio

**Comment by FJBDev** [comment]
@QQ7ita 

You can see [here](https://github.com/mytourbook/mytourbook/blob/main/bundles/net.tourbook.export/src/net/tourbook/export/fit/FitSportMapper.java#L28) more details on how the FIT sport type is mapped from the tour type (MT uploads each tour to Strava as a FIT file). 
And you can see more details [here](https://github.com/mytourbook/mytourbook/issues/845#issuecomment-1515472101) where I discussed it in the FIT export feature.

My hunch is that since you use MT in italian, the mapping function doesn't recognize the italian tour type names and therefore uses the `GENERIC` FIT sport type

I am up to improve the mapping function but I am not sure how to take into account any other languages (italian, french german...)

**Reply by QQ7ita** [reply]
Ok, I can try to use MT in english and check if it works well.
I will let you know.

**Reply by FJBDev** [reply]
let me know of your finding, I don't think it's necessary to change MT's language, only the tour type must be in English.

Another idea I had was to ask to translate each Garmin Sport Type from each language supported in MT (italian, czech, german etc...) so that people can continue using their tour types list in their favorite language.

Thoughts ?

**Comment by QQ7ita** [comment]
As you suggested using MT in English is not needed. 
I changed the description of the activity type by putting "cycling" and now it works.
I had renamed the categories BDC (for road bike) and MTB (for mountain bike), now it will be CYCLING and E_BIKE_MOUNTAIN.

**Reply by FJBDev** [reply]
Thanks for confirming.
I am glad to know it's working but i still think it would be nice to support any languages
